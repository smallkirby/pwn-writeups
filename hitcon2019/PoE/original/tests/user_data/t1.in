i 3 1234
i 0 abcd
i 3 1234
d 0 8
d 4 5
d 4 4
n
i 0 ABCD
n
i 0 9876
s 1
d 0 4
s 2
d 1 1
s 3
d 0 4
s 0
d 0 4
