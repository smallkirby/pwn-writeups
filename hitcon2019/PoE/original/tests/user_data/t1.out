Done.
Done.
Done.
abc1234d
234d
Switched to new tab 1.
Done.
Switched to new tab 2.
Done.
Switched to tab 1.
ABCD
Switched to tab 2.
8
Switched to tab 2.
9876
Switched to tab 0.
abc1
